rasa/core/channels/console.py

```diff
@@ -50,14 +58,20 @@ def print_bot_output(message, color=rasa.cli.utils.bcolors.OKBLUE):
         )
 
 
-def get_cmd_input():
-    response = questionary.text(
-        "", qmark="Your input ->", style=Style([("qmark", "#b373d6"), ("", "#b373d6")])
-    ).ask()
+def get_cmd_input(button_question: questionary.Question):
+    if button_question:
+        text = button_question.ask()
+        response = text[text.find("(") + 1 : text.find(")")]  # get intent
```

Comment:
Can we extract that into a separate method as the exact code is also used in interactive learning?